src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -83,6 +76,83 @@ class IteratorExpr:
     indices: dict[gtx_common.Dimension, ValueExpr]
 
 
+class DataflowInputEdge(Protocol):
+    """Allows to setup an input edge in a map scope."""
+
+    @abc.abstractmethod
+    def connect(self, me: dace.nodes.MapEntry) -> None:
+        pass
```

Comment:
I think the coding guide says that you have to use `...` here, but I am not sure.

Reply:
I looked it up, according to the first point of [this section](https://github.com/GridTools/gt4py/blob/main/CODING_GUIDELINES.md#common-questions) it needs to be `...`.